projects/exui-common-lib/src/lib/components/checkbox-list/checkbox-list.component.ts

```diff
@@ -62,6 +65,11 @@ export class CheckboxListComponent<T> implements OnChanges {
     this.checkSelection();
   }
 
+  // checks the data projected into the component
+  public ngAfterContentChecked(): void {
+    this.ref.detectChanges();
+  }
+
```

Comment:
is this additional change necessary?

Reply:
required by the rpxTranslationPipe tests.